src/mmw/mmw/middleware.py

```diff
@@ -0,0 +1,40 @@
+from django.conf import settings
+from django.core.handlers.base import BaseHandler
+from django.core import urlresolvers
+
+
+def bypass_middleware(view):
+    view.bypass_middleware = True
+
+    return view
+
+
+class BypassMiddleware(object):
```

Comment:
One question is what is the type of value for `object` here?  Can it be removed or renamed to reflect its content, doesn't look like it's used at all?